react/features/recording/components/LiveStream/native/StreamKeyPicker.js

```diff
@@ -67,10 +67,22 @@ class StreamKeyPicker extends Component<Props, State> {
     render() {
         const { broadcasts } = this.props;
 
-        if (!broadcasts || !broadcasts.length) {
```

Comment:
@zbettenbuk, is it preferred that nothing be shown if broadcasts are empty?

Reply:
It was in the past, but now your changes make more sense